packages/integration_test/README.md

```diff
@@ -154,7 +154,7 @@ Future<void> main() async {
 ```dart
 void main() {
   final IntegrationTestWidgetsFlutterBinding binding =
-      IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+      IntegrationTestWidgetsFlutterBinding();
```

Comment:
```suggestion
      IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

Reply:
Same as above, the type declaration can be removed I think:

```dart
final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized() as IntegrationTestWidgetsFlutterBinding;
```

Reply:
Have updated it